FLEDGE.md

```diff
@@ -365,10 +365,8 @@ The arguments to this function are:
 
     ```
     { 'topWindowHostname': 'www.example-publisher.com',
-      'interestGroupOwner': 'www.example-dsp.com',
-      'interestGroupName': 'womens-running-shoes',
```

Comment:
Hmm, it seems like we still should report the IG name to the *buyer*, whose IG it is, right?  Fine removing it from the reporting function for the seller, though.

Reply:
Oops, added explicitly the buyer signals - that's one of the problems of sharing code, particularly without unit tests.  We are providing more than enough info to the buyer to break k-anonymity of info from the IG, but that's an issue for another time.